packages/core/src/audio/AudioManager.ts

```
@@ -3,48 +3,114 @@
+  private static _context: AudioContext | null = null;
+  private static _gainNode: GainNode | null = null;
   private static _isResuming = false;
+  private static _isAudioAvailable = true;
+  private static _isWebKit = false;
 
-  static getContext(): AudioContext {
+  static getContext(): AudioContext | null {
+    if (!AudioManager._isAudioAvailable) {
+      return null;
+    }
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)